src/Sulu/Bundle/ContactBundle/Util/IdsHandlingTrait.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Util;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+/**
+ * Provides function to sort array by ids array.
+ */
+trait IdsHandlingTrait
+{
+    /**
+     * @var PropertyAccessor
+     */
+    private $accessor;
+
+    /**
+     * Returns accessor.
+     *
+     * @return PropertyAccessor
+     */
+    private function getAccesssor()
+    {
+        if (null === $this->accessor) {
+            $this->accessor = PropertyAccess::createPropertyAccessor();
```

Comment:
I think I won't use the `PropertyAccessor`... It makes the code harder to read and debug.